src/scripts/main.js

```diff
@@ -2,7 +2,39 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tablElement = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  const trElement = document.createElement('tr');
+  const thName = document.createElement('th');
+
+  thName.textContent = person.name;
+  trElement.append(thName);
+
+  const thSex = document.createElement('th');
+
+  thSex.textContent = person.sex;
+  trElement.append(thSex);
+
+  const thBorn = document.createElement('th');
+
+  thBorn.textContent = person.born;
+  trElement.append(thBorn);
+
+  const thDied = document.createElement('th');
+
+  thDied.textContent = person.died;
+  trElement.append(thDied);
+
+  const thAge = document.createElement('th');
+
+  thAge.textContent = person.died - person.born;
+  trElement.append(thAge);
+
+  const thCentury = document.createElement('th');
+
+  thCentury.textContent = Math.ceil(person.died / 100);
+  trElement.append(thCentury);
+
+  tablElement.append(trElement);
```

Comment:
you are doing the same action several times. think how you can avoid repeating and shorty the code. Maybe would be easier to use `insertAdjacentHTM`L